custom_components/myskoda/device_tracker.py

```diff
@@ -84,8 +84,13 @@ def longitude(self) -> float | None:  # noqa: D102
     def extra_state_attributes(self) -> dict:
         """Return extra state attributes."""
         attributes = {}
-        if render := self.get_renders().get("main"):
-            attributes["entity_picture"] = render
+
+        render_list = self.get_composite_renders().get("unmodified_exterior_front")
+        if isinstance(render_list, list) and render_list:
+            for render in render_list:
+                if isinstance(render, dict) and "exterior_front" in render:
```

Comment:
In the info return for my Suprb iV I see this:
```
                    {
                        "layers": [
                            {
                                "url": "https://skodavinimagecache.blob.core.windows.net/cache3v3s20201119/0F0FEKAcM-837-icSUg3f_sXOJRZCyGa-CK2vH0E59-4A6othXxnP8dOzuUaqJV-H9AsCqv16YbLX2EKOzn5fjwk-IaBMLRAHXUTFO-19201080dayvext_front1080.png",
                                "type": "REAL",
                                "order": 0,
                                "view_point": "exterior_front"
                            },
                            {
                                "url": "https://mspgwlivestorage.blob.core.windows.net/renders/cables/superb_3V_cable_charging_light_v1.png",
                                "type": "REAL",
                                "order": 1,
                                "view_point": "exterior_front"
                            }
                        ],
                        "view_type": "CHARGING_LIGHT"
                    },
```
So there can be list where there are two `exterior_front` entries and the second one is just a plug.
Of course, currently most probably it selects the one from `UNMODIFIED_EXTERIOR_FRONT` view type (where there is only one entry), but I think there should be some better selection for the URL than just `exterior_front` key.

Reply:
`render_list = self.get_composite_renders().get("unmodified_exterior_front")` ==> this selects UNMODIFIED_EXTERIOR_FRONT

 and this
```
if isinstance(render_list, list) and render_list:
            for render in render_list:
                if isinstance(render, dict) and "exterior_front" in render:
```
will go trough all urls and selects the one with "exterior_front" from UNMODIFIED_EXTERIOR_FRONT ;)

Reply:
missed that one :)